config/templates/logging/secret.yaml

```
@@ -0,0 +1,17 @@
+# Copyright KubeArchive Authors
+# SPDX-License-Identifier: Apache-2.0
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kubearchive-logging
+  namespace: kubearchive
+  labels:
+    app.kubernetes.io/name: kubearchive-logging
+    app.kubernetes.io/component: logging
+    app.kubernetes.io/part-of: kubearchive
+    app.kubernetes.io/version: "${NEXT_VERSION}"
+type: Opaque
+data:
+  password: Y2hhbmdlbWU=  # notsecret
+  user: YWRtaW4=
```

This need to be uppercase:
```suggestion
  PASSWORD: Y2hhbmdlbWU=  # notsecret
  USER: YWRtaW4=
```